ingestions/ingest_csvs.py

```diff
@@ -136,15 +138,20 @@ def main(argv=None):
 
     # assign input arguments.
     parser.add_argument("-c", "--csvfile", dest="csvfile", type=Path, required=True)
-    parser.add_argument("-t", "--ingest_type", dest="ingest_type", type=str, choices=('recovered', 'telemetered'),
+    parser.add_argument("-t", "--ingest_type", dest="ingest_type", type=str, 
+                        choices=('recovered', 'telemetered', 'stage', 'mock'), 
```

Comment:
Stage and mock should be options for "state" within the ingest request. Options for type are recovered, telemetered, and playback. Options for state are stage, mock, run.

Reply:
OK, these have been properly split out, with run set as the default state.